nginx_conf

```diff
@@ -0,0 +1,44 @@
+server {
+    listen 443 ssl;
+    listen [::]:443 ssl;
+    include snippets/self-signed.conf;
+    include snippets/ssl-params.conf;
+
+
+    server_name 192.168.50.2;
+    root /var/www/wordpress;
+
+    index index.html index.htm index.php;
+
+    location / {
+        #try_files $uri $uri/ =404;
+         try_files $uri $uri/ /index.php$is_args$args;
+    }
+
+    location ~ \.php$ {
+        include snippets/fastcgi-php.conf;
+        fastcgi_pass unix:/var/run/php/php7.4-fpm.sock;
+    }
+
+    location ~ /\.ht {
+        deny all;
+    }
+
+    location = /favicon.ico { log_not_found off; access_log off; }
+    location = /robots.txt { log_not_found off; access_log off; allow all; }
+    location ~* \.(css|gif|ico|jpeg|jpg|js|png)$ {
+        expires max;
+        log_not_found off;
+    }
+
+}
+
+server {
+    listen 80;
+    listen [::]:80;
+
+    server_name 192.168.50.2;
+
+    return 302 https://$server_name$request_uri;
```

Comment:
why here return code is 302 and not 301?

Reply:
I'll make it permenant

Reply:
I'm not saying that it's wrong. I'm just wondering why do you use it in this case, same question would be if you use `301`.

Reply:
I thought I could redirect the HTTP traffic to HTTPS

Reply:
redirect is a great idea. You can redirect with status `302` and `301` so why `302` ;)